src/GovUk.Education.ExploreEducationStatistics.Data.Services/SubjectCsvMetaService.cs

```diff
@@ -89,6 +89,7 @@ public class SubjectCsvMetaService : ISubjectCsvMetaService
                 .SingleAsync(
                     predicate: rf =>
                         rf.File.SubjectId == releaseSubject.SubjectId
+                        && rf.File.Type == FileType.Data
```

Comment:
Might be worth having a test with a data and meta file to check that this doesn't regress?

Reply:
Added one extra test `GetSubjectCsvMeta_Headers_MultipleReleaseFilesExist`